.github/workflows/python.yml

```
@@ -42,7 +42,7 @@ jobs:
         run: pip -v install dist/*.tar.gz
 
       - name: Test import
-        run: python -c 'import idyntree.bindings'
+        run: python -c 'import idyntree.swig'
```

To test that import.bindings works fine (it should import swig by defaults, unless I am did not get the discussion in the PR), perhaps we should leave the test as it is? 
```suggestion
        run: python -c 'import idyntree.bindings'
```